integration-tests/tests/analysis-context-bar.js

```diff
@@ -15,16 +15,26 @@ const testAnalysisContextBarFn = _.flow(
   // Navigate to appropriate part of UI (the analysis tab)
   await performAnalysisTabSetup(page, token, testUrl, workspaceName)
 
+  // Ensure UI displays the runtime Terminal icon is present + disabled
+  const tooltipTextTerminal = 'Terminal'
+  await findElement(page, clickable({ textContains: tooltipTextTerminal, isEnabled: false }))
```

Comment:
Why is it necessary and how is it related to the [error on the QA ticket](https://broadworkbench.atlassian.net/browse/QA-1903) to "Ensure UI displays the runtime Terminal icon is present + disabled"?

Reply:
By `UI correctness`, do you mean `UI readiness`? My assumption reading the change was that the change tries to ensure that

```javascript
await click(page, clickable({ textContains: 'Environment Configuration' }))
```

doesn't happen too soon, by first checking for another element (`Terminal`) that shows up on the same page as `Environment Configuration` does. If my assumption is correct, I'd like to understand why checking for `Environment Configuration` isn't sufficient. 

Or was it that you just wanted to add another check to make sure `Terminal` is also there?

Reply:
Test checks the Terminal is enabled at L33: `await findElement(page, clickable({ textContains: tooltipTextTerminal, isEnabled: true }), { visible: true })` after create runtime. What if the Terminal was already enabled before Create Runtime? Is that a bug or still correct in UI?

Reply:
Gotcha  👍🏼